### ISSUE TITLE
Missing 64bit build for desktop version

### ISSUE THREAD

**Issue by clarjon1** [PhieF/CarnetDocumentation#81]
Heads up, the 64bit build for the latest version is missing;

From https://qn.phie.ovh/binaries/desktop/daily/
![image](https://user-images.githubusercontent.com/627845/69098515-b9c5bb80-0a26-11ea-8da1-8f8c8b6509d0.png)

**Comment by PhieF** [comment]
Yes there was a pb with the 0.18.7 but the current version is there

https://qn.phie.ovh/binaries/desktop/current64.AppImage
